pkg/validation/app.go

```diff
@@ -266,6 +283,19 @@ func (v *Validator) validateLabels(ctx context.Context, cr v1alpha1.App) error {
 	return nil
 }
 
+func (v *Validator) validateLabelsOrgApp(ctx context.Context, cr v1alpha1.App) error {
```

Comment:
```suggestion
func (v *Validator) validateOrgLabels(ctx context.Context, cr v1alpha1.App) error {
```

To align with the other name.